File: app/src/main/java/ru/semper_viventem/chromeor/presentation/view/about/AboutActivity.java
package ru.semper_viventem.chromeor.presentation.view.about;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.google.android.gms.analytics.HitBuilders;

import ru.semper_viventem.chromeor.R;
import ru.semper_viventem.chromeor.presentation.view.main.MainActivity;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        MainActivity.mTracker.setScreenName("About");
        MainActivity.mTracker.send(new HitBuilders.ScreenViewBuilder().build());
    }
}

File: app/src/main/kotlin/ru/semper_viventem/chromeor/presentation/view/about/AboutActivity.kt
package ru.semper_viventem.chromeor.presentation.view.about

import android.os.Bundle
import com.arellomobile.mvp.MvpAppCompatActivity
import com.arellomobile.mvp.presenter.InjectPresenter
import ru.semper_viventem.chromeor.R
import ru.semper_viventem.chromeor.presentation.presenter.AboutPresenter

class AboutActivity : MvpAppCompatActivity(), AboutView {

    @InjectPresenter
    lateinit var mAboutPresenter: AboutPresenter

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)

        mAboutPresenter.trackerOpenActivity()
    }
}
